server/store/string_store.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2016 The Chihaya Authors. All rights reserved.
+// Use of this source code is governed by the BSD 2-Clause license,
+// which can be found in the LICENSE file.
+
+package store
+
+import "fmt"
+
+// PrefixInfohash is the prefix to be used for infohashes.
+const PrefixInfohash = "ih-"
```

Comment:
The idea was to include the most common prefixes here, to avoid collisions. Store dependent middleware has to use the store anyway, so all store dependent middlewares have access to this.